src/app/panels/righthand-panel.js

```diff
@@ -2,29 +2,31 @@ var yo = require('yo-yo')
 var remixLib = require('remix-lib')
 var EventManager = remixLib.EventManager
 var TabbedMenu = require('../tabs/tabbed-menu')
-var compileTab = require('../tabs/compile-tab')
-var runTab = require('../tabs/run-tab')
-var settingsTab = require('../tabs/settings-tab')
-var analysisTab = require('../tabs/analysis-tab')
-var debuggerTab = require('../tabs/debugger-tab')
-var supportTab = require('../tabs/support-tab')
-var pluginTab = require('../tabs/plugin-tab')
+var CompileTab = require('../tabs/compile-tab')
+var RunTab = require('../tabs/run-tab')
+var SettingsTab = require('../tabs/settings-tab')
+var AnalysisTab = require('../tabs/analysis-tab')
+var DebuggerTab = require('../tabs/debugger-tab')
+var SupportTab = require('../tabs/support-tab')
+var PluginTab = require('../tabs/plugin-tab')
 var PluginManager = require('../../pluginManager')
 
 var css = require('./styles/righthand-panel-styles')
 
-function RighthandPanel (appAPI, events, opts) {
+function RighthandPanel (api = {}, events = {}, opts = {}) {
```

Comment:
please keep using `appAPI` here

Reply:
the idea is to change every module signature to `(api = {}, events = {}, opts = {})` according to your PR to update best practices.
Personally I think it matters that the arguments are named the same in every module, so that people get used to rely on `api`, `events` and `opts` and not have different name in each module.

I see this PR as an opportunity to come closer to that goal